webapp/components/features/FollowList/FollowList.vue

```diff
@@ -0,0 +1,165 @@
+<template>
+  <base-card class="follow-list">
+    <template v-if="connections && connections.length">
+      <h5 class="title spacer-x-small">
+        {{ userName | truncate(15) }} {{ $t(`profile.network.${type}`) }}
+      </h5>
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-03-31T12:00:41Z" title="Tuesday, March 31st 2020, 2:00:41 pm +02:00">Mar 31, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -0,0 +1,133 @@
+<template>
+  <base-card class="follow-list">
+    <template v-if="connections && connections.length">
+      <ds-text tag="h5" color="soft" class="spacer-x-small">
+        {{ userName | truncate(15) }} {{ $t(`profile.network.${type}`) }}
+      </ds-text>
```


let's refactor out all styleguide components? you can use a `h5` tag here and apply `CSS` to make it soft, etc...
and if you give this a class of `title`, for example, you can target it with `CSS` like: 
```
.follow-list {
  > .title {
   ....
  }
}
```
which says only target the class title that is a direct child of `follow-list`

Reply:
> [<img alt="rbeer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/rbeer) **Authored by [rbeer](https://github.com/rbeer)**
_<time datetime="2020-04-01T05:06:30Z" title="Wednesday, April 1st 2020, 7:06:30 am +02:00">Apr 1, 2020</time>_

---


Was just hesitating to do that. So basically, whenever I come across a ds-* tag, refactor it out?
The question is then, what design-tokens do I use? Just any that fit and if nothing applies, create one? We don't want to end up with 20 definitions of e.g. a card-heading is what I'm concerned about. @alina-beck 

Anyway, there is a ds-input later on, I'll start with by turning it into a base component.

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-04-01T06:00:28Z" title="Wednesday, April 1st 2020, 8:00:28 am +02:00">Apr 1, 2020</time>_

---


yes, please. There should be design tokens for all the sizes that are used in `ds-text`... otherwise maybe see if one of them looks good, ie not too big or too small.
As far as the `ds-input`, I would probably leave that because it would actually involve migrating the `ds-input` component and refactoring, ie separate ticket(code creep).
My larger point was that if it's possible to accomplish the same with `html` tags and `CSS`, let's do that, so we don't make it harder on ourselves to refactor later.

Reply:
> [<img alt="rbeer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/rbeer) **Authored by [rbeer](https://github.com/rbeer)**
_<time datetime="2020-04-01T09:28:54Z" title="Wednesday, April 1st 2020, 11:28:54 am +02:00">Apr 1, 2020</time>_

---


done in 72dfeb3